pkgs/applications/version-management/gitea/static-root-path.patch

```diff
@@ -2,12 +2,12 @@ diff --git i/modules/setting/setting.go w/modules/setting/setting.go
 index aafe2d1b..1e4a8064 100644
 --- i/modules/setting/setting.go
 +++ w/modules/setting/setting.go
-@@ -683,7 +683,7 @@ func NewContext() {
+@@ -730,7 +730,7 @@ func NewContext() {
  	LocalURL = sec.Key("LOCAL_ROOT_URL").MustString(defaultLocalURL)
  	OfflineMode = sec.Key("OFFLINE_MODE").MustBool()
  	DisableRouterLog = sec.Key("DISABLE_ROUTER_LOG").MustBool()
--	StaticRootPath = sec.Key("STATIC_ROOT_PATH").MustString(workDir)
+-	StaticRootPath = sec.Key("STATIC_ROOT_PATH").MustString(AppWorkPath)
 +	StaticRootPath = sec.Key("STATIC_ROOT_PATH").MustString("@data@")
- 	AppDataPath = sec.Key("APP_DATA_PATH").MustString("data")
+	AppDataPath = sec.Key("APP_DATA_PATH").MustString(path.Join(AppWorkPath, "data"))
```

Comment:
Should not this instance of `AppWorkPath` be replaced by `@data@` too?

Reply:
Is `APP_DATA_PATH` supposed to be writeable? In that case it should be not in the nix store.

Reply:
I haven't had a chance to test this yet, but the original understanding behind putting the data in the store is the nixos service copies that data from the nix store -> the state directory before starting the service. I believe this was related to the locales and static files that needed to be in the state directory.

Reply:
sorry @orivej I didn't see your comment. I'll submit a PR to adjust that as well.

Reply:
Err wait, nevermind. AppDataPath we don't want in the store. The whole point for replacing StaticRootPath was because it writes those files in the build there. This is correct as-is unless someone disagrees.